lib/modules/platform/github/index.ts

```diff
@@ -655,14 +656,20 @@ export async function getBranchPr(branchName: string): Promise<Pr | null> {
     autoclosedPr?.title?.endsWith(' - autoclosed') &&
     autoclosedPr?.closedAt
   ) {
-    const closedMillisAgo = DateTime.fromISO(autoclosedPr.closedAt)
+    const closedMillisAgo = DateTime.fromISO(autoclosedPr.closedAt, {
+      zone: 'utc',
```

Comment:
Is this related to the current problem, or just something you fixed at the same time? And is it a fix - i.e. something wasn't working before, including tests - or just more robust code?

Reply:
was supposed to be the later one, but actually it is not needed.
`toMillis` is millis since epoch which is already utc. 
ill revert this